diff --git a/debian/control b/debian/control
--- a/debian/control
+++ b/debian/control
@@ -82,11 +82,11 @@ Package: soapysdr-module-limesuiteng
 Section: libs
 Architecture: all
 Pre-Depends: ${misc:Pre-Depends}
-Depends: soapysdr0.8-module-limesuiteng, ${misc:Depends}
+Depends: soapysdr0.7-module-limesuiteng, ${misc:Depends}
 Description: Lime Suite NG - SoapySDR bindings metapackage
  Lime Suite NG application software.
 
-Package: soapysdr0.8-module-limesuiteng
+Package: soapysdr0.7-module-limesuiteng
 Section: comm
 Architecture: any
 Multi-Arch: same
