diff --git a/debian/control b/debian/control
--- a/debian/control
+++ b/debian/control
@@ -72,13 +72,6 @@ Description: Lime Suite NG - Install firmware and gateware images
  This is an empty dependency package that pulls in the images
  for the default version of Lime Suite NG.
 
-Package: limesuiteng-litepcie-kernel-module
-Section: libs
-Architecture: any
-Multi-Arch: same
-Depends: ${shlibs:Depends}, ${misc:Depends}
-Description: Lime Suite NG LitePCIe kernel driver.
-
 Package: soapysdr-module-limesuiteng
 Section: libs
 Architecture: all
